### PR TITLE
Small lint changes

### PR DESCRIPTION
**Pull request by DoctorDib**
This branch is merged from the other other PR, so please do this one last.

Changes:
 - Small lint changes as well as changing the package.json commands that should be ran before you commit.

### CODE REVIEW

File: client/src/Common/Helpers/DB_Helpers.tsx [hunk 1]
```diff
@@ -12,9 +12,6 @@ export const sendPost = async (
     postAttributes: IPostAttributes,
     postType: number,
 ): Promise<boolean> => {
-    const postAttributesSetup: IPostAttributes = postAttributes;
```
Comment (BenRGH) [inline client/src/Common/Helpers/DB_Helpers.tsx]: ESlint complaint here about needing a new line

File: client/src/Common/Helpers/DB_Helpers.tsx [hunk 2]
```diff
@@ -85,13 +82,25 @@ export const updateVehicleStat = async (vehicleID: number, newStat: number): Pro
...
 };
 
-export const addNewVehicle = async (ownerID: number, data: IInputFields): Promise<boolean> => {
-    const cleanData = checkSQLInObject(data);
+const stripData64 = (images: Array<IImageSettings>) =>
+    images.map((data) => ({
+        name: data.name,
+        is_main_image: data.is_main_image,
+        type: data.type,
+        crop: {
```
Comment (BenRGH) [inline client/src/Common/Helpers/DB_Helpers.tsx]: Eslint complaint here about not needing to be on a new line

**Comment by BenRGH** [comment]
Pull master and then push an update to this before I continue

File: client/src/Common/Helpers/DB_Helpers.tsx (hunk 1 above)
Comment (BenRGH) [reply client/src/Common/Helpers/DB_Helpers.tsx]: Github has decided this comment shouldn't be anywhere near where I set it

File: client/src/Common/Helpers/DB_Helpers.tsx (hunk 2 above)
Comment (BenRGH) [reply client/src/Common/Helpers/DB_Helpers.tsx]: this refers to the images.map

File: client/src/Components/ImageCropper/index.tsx [hunk 3]
```diff
@@ -1,6 +1,4 @@
-import React, {
-    useState, useCallback, useRef, useEffect,
-} from 'react';
+import React, { useState, useCallback, useRef, useEffect } from 'react';
```
Comment (BenRGH) [inline client/src/Components/ImageCropper/index.tsx]: eslint complaint on this first line

File: client/src/Components/ImageCropper/index.tsx [hunk 4]
```diff
@@ -81,17 +75,23 @@ const ImageCropped = (props: IImageCropProps): React.Element<IImageCropProps> =>
...
     useEffect(() => {
-        if (!completedCrop || !previewCanvasRef.current || !imgRef.current) { return; }
+        if (!completedCrop || !previewCanvasRef.current || !imgRef.current) {
+            return;
+        }
 
         const image = imgRef.current;
         const canvas = previewCanvasRef.current;
-        if (image === null || canvas === null) { return; }
+        if (image === null || canvas === null) {
```
Comment (BenRGH) [inline client/src/Components/ImageCropper/index.tsx]: You already check this above?

File: client/src/Components/ImageCropper/index.tsx [hunk 5]
```diff
@@ -139,17 +139,11 @@ const ImageCropped = (props: IImageCropProps): React.Element<IImageCropProps> =>
                     className={classes.button}
                 >
                     {' '}
-Confirm
-                    {' '}
+                    Confirm{' '}
```
Comment (BenRGH) [inline client/src/Components/ImageCropper/index.tsx]: eslint complaints about new lines with these {' '}

File: client/src/Components/ImageUploader/index.tsx [hunk 6]
```diff
@@ -3,9 +3,7 @@ import {
     Paper, IconButton, CardMedia, Backdrop,
 } from '@material-ui/core';
 import { SpeedDialAction, SpeedDial, SpeedDialIcon } from '@material-ui/lab';
-import {
-    Add, Crop, Clear, StarBorder,
-} from '@material-ui/icons';
+import { Add, Crop, Clear, StarBorder } from '@material-ui/icons';
```
Comment (BenRGH) [inline client/src/Components/ImageUploader/index.tsx]: Another eslint complaint on these imports

File: client/src/Components/ImageUploader/index.tsx [hunk 7]
```diff
@@ -78,26 +84,34 @@ const ImageUploader = (props:IImageUploaderProps): React.ReactElement<IImageUplo
...
 
         newImages = moveItemInArray(newImages, chosenIndex, 0);
 
         setImages(newImages);
         handleClose();
     };
     const mapImages = images.map((image: IImageSettings) => (
-        <Paper key={image.id} className={classes.container} style={{ border: image.is_main_image ? '3px solid rgb(204, 204, 4)' : '0' }}>
-
+        <Paper
```
Comment (BenRGH) [inline client/src/Components/ImageUploader/index.tsx:112]: Eslint doesn't like this linebreak

Comment (BenRGH) [reply client/src/Components/ImageUploader/index.tsx:112]: Github messing up again, this is for line 137: onclick

File: client/src/Components/Posts/index.tsx [hunk 8]
```diff
@@ -118,7 +118,8 @@ const Forum = (props: IForumProps): React.ReactElement<IForumProps> => {
         }
     };
 
-    const commentConfirmationClick = (comment: IComment, response: boolean) => (() => onVehicleConfirm(comment, response));
+    const commentConfirmationClick = (comment: IComment, response: boolean) => () =>
+        onVehicleConfirm(comment, response);
```
Comment (BenRGH) [inline client/src/Components/Posts/index.tsx]: eslint linebreak complaint

File: client/package.json [hunk 9]
```diff
@@ -11,10 +11,10 @@
   "scripts": {
     "start": "snowpack dev",
     "build": "snowpack build",
-    "fix": "eslint \"src/**/*.{js,jsx,ts,tsx}\" --fix",
     "test": "jest",
-    "format": "prettier --write \"src/**/*.{js,jsx,ts,tsx}\"",
-    "lint": "prettier --check \"src/**/*.{js,jsx,ts,tsx}\""
+    "format" : "npm run format:eslint && npm run format:prettier",
+    "format:eslint": "eslint \"src/**/*.{js,jsx,ts,tsx}\" --fix",
```
Comment (BenRGH) [inline client/package.json:16]: I think we should avoid the use of the : character, could cause issues in certain terminals. Maybe use - instead

Comment (DoctorDib) [reply client/package.json:16]: That's widely used when it comes to running two commands under one npm, which is why I went with it

Comment (BenRGH) [reply client/package.json:16]: So long as it doesn't cause issue if someone decides to use a weird terminal or a mac

**Pull request merged**